app/javascript/comment.vue

```diff
@@ -85,6 +85,11 @@ export default {
       const tribute = new Tribute({ collection: collection })
       tribute.attach(textareas)
     })
+
+    this.$nextTick( () => {
+      const commentAnchor = location.hash;
+      commentAnchor ? location.href = location.href : null;
```

Comment:
三項演算子は戻り値を使う場合に利用するケースが多いですね。
今回のだと下記で良さそうです。

```suggestion
      if (commentAnchor) {
        location.href = location.href
      }
```

Reply:
そもそもlocation.hash.length == 0ならnextTickの処理自体が不要ですね。

Reply:
@bluerabbit ご指摘ありがとうございます！！
> 三項演算子は戻り値を使う場合に利用する

使いどころがよくわかっていませんでした💦
if文で書き直しますー

> そもそもlocation.hash.length == 0ならnextTickの処理自体が不要

そうですね・・・！！
nextTickの処理はifの中に入れ込みます！